sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -450,16 +450,18 @@ public <T> CosmosPagedIterable<T> readAllItems(
 
     /**
      * Reads an item in the current container.
-     * <br/>
-     * This operation is used to retrieve a single item from a container based on its unique identifier (ID) and partition key.
-     * The readItem operation provides direct access to a specific item using its unique identifier, which consists of the item's ID and the partition key value. This operation is efficient for retrieving a known item by its ID and partition key without the need for complex querying.
-     * 
-     * @param <T> the type parameter.
-     * @param itemId the item id.
-     * @param partitionKey the partition key.
-     * @param itemType the class type of item.
+     *
+     * @param <T> the type parameter. Required.
+     * @param itemId the item id. Required.
+     * @param partitionKey the partition key. Required.
+     * @param itemType the class type of item. Required.
      * @return the Cosmos item response.
-     * @throws com.azure.cosmos.implementation.NotFoundException if document with the specified itemId does not exist
+     *
+     * @exception com.azure.cosmos.implementation.NotFoundException When there is no item in the database with a given itemId
```

Comment:
I don't think it will throw `NotFoundException` directly (because it is implementation detail), rather wrapped around `CosmosException`. Is that not true?